assets/pages/paper-subscription-landing/components/form.jsx

```diff
@@ -5,45 +5,91 @@
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 
-import { type PaperBillingPlan } from 'helpers/subscriptions';
+import { type Option } from 'helpers/types/option';
+import { getNewsstandSaving, getNewsstandPrice, type PaperBillingPlan } from 'helpers/subscriptions';
 import { type Action } from 'components/productPage/productPagePlanForm/productPagePlanFormActions';
 import ProductPagePlanForm, { type StatePropTypes, type DispatchPropTypes } from 'components/productPage/productPagePlanForm/productPagePlanForm';
 
 import { type State } from '../paperSubscriptionLandingPageReducer';
 import { setPlan } from '../paperSubscriptionLandingPageActions';
 
 
+// ---- Helpers ----- //
+
+const getPriceStr = (price: Option<number>): Option<string> =>
```

Comment:
I think you can use the `displayPrice` function from subscriptions.js here

Reply:
I can't as these prices are fed from a different place, `displayPrice` uses the hardcoded list in the helper

Reply:
Ok, let's leave it for a later refactor once the price stuff is better integrated 👍